src/index.ts

```diff
@@ -41,7 +23,7 @@ async function main() {
 
 	for (const store of Stores) {
 		Logger.debug(store.links);
-		setTimeout(tryLookupAndLoop, getSleepTime(), browser, store);
+		setTimeout(await tryLookupAndLoop, getSleepTime(), browser, store);
```

Comment:
`await` cannot be used here, at the very least it won't work.

Reply:
I thought the same, which I thought was weird. It doesn't complain either way. I wonder if `tsc` throws it out...

Reply:
Oh, the linter catches it though!